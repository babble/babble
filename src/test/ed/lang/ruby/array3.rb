

a = [ 1 , 2 , 3 ].collect{ |z| z + 1 }
puts a.length
puts a[0]
puts a[2]

puts a.join( "|" )


a = [ 1 , 2 , 3 ].collect!{ |z| z + 1 }
puts a.length
puts a[0]
puts a[2]

puts a.join( "|" )

a = [ 1 , 2 , 3 ]
puts a[0]
a.collect!{ |z| z + 1 }
puts a.length
puts a[0]
puts a[2]

puts a.join( "|" )
