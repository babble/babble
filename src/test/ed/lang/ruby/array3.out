3
2
4
2|3|4
3
2
4
2|3|4
1
3
2
4
2|3|4
2
3
2
---
1
2
2
3
***
67
0
C
69
0
E
75
8218
K
.
C

.
E

.
K
&8218;
C,,E,,K,&8218;
